packages/sirius-web/frontend/sirius-web-application/src/modals/update-library/UpdateLibraryModal.tsx

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import DialogTitle from '@mui/material/DialogTitle';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import Button from '@mui/material/Button';
+import DialogActions from '@mui/material/DialogActions';
+import { UpdateLibraryTable } from './UpdateLibraryTable';
+import { useState } from 'react';
+import { UpdateLibraryModalProps, UpdateLibraryModalState } from './UpdateLibraryModal.types';
+import { useUpdateLibrary } from './useUpdateLibrary';
+import { useCurrentProject } from '../../views/edit-project/useCurrentProject';
+
+export const UpdateLibraryModal = ({ open, title, namespace, name, version, onClose }: UpdateLibraryModalProps) => {
+  const [state, setState] = useState<UpdateLibraryModalState>({
+    selectedLibraries: [],
+  });
+
+  const { updateLibrary } = useUpdateLibrary();
+
+  const onSelectionChange = (selection) => {
```

Comment:
Let's not use the concept of selection in the naming convention to reference anything else than the workbench selection.

Reply:
Done